modules/oc-adm-by-example-content.adoc

```diff
@@ -346,6 +350,96 @@ Display and filter node logs
 
 
 
+== oc adm ocp-certificates monitor-certificates
+Watch platform certificates.
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Watch platform certificates.
+  oc adm ocp-certificates monitor-certificates
+----
+
+
+
+== oc adm ocp-certificates regenerate-leaf
+Regenerate client and serving certificates of an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate all known leaf certificates on the cluster.
+  oc adm certificates regenerate-leaf -A --all
+  
+  # Regenerate a leaf certificate contained in a particular secret.
+  oc adm ocp-certificates regenerate-leaf -n openshift-config-managed kube-controller-manager-client-cert
```

Comment:
the command is incorrect , should be like :
`oc adm ocp-certificates regenerate-leaf -n openshift-config-managed kube-controller-manager-client-cert
error: the server doesn't have a resource type "kube-controller-manager-client-cert"`
`oc adm ocp-certificates regenerate-leaf -n openshift-config-managed  secret default-token-sfmqr`

Reply:
Removing this example temporarily